lib/render/VolumeIsoRenderer.cpp

```diff
@@ -37,7 +37,10 @@ VolumeIsoRenderer::VolumeIsoRenderer( const ParamsMgr*    pm,
     // An ugly fix but I don't think we have a mechanism for this
     if (_needToSetDefaultAlgorithm()) {
         VolumeParams *vp = (VolumeParams*)GetActiveParams();
-        Grid *grid = _dataMgr->GetVariable(vp->GetCurrentTimestep(), vp->GetVariableName(), vp->GetRefinementLevel(), vp->GetCompressionLevel());
+		vector <double> minExt, maxExt;
+		vp->GetBox()->GetExtents(minExt, maxExt);
+
+        Grid *grid = _dataMgr->GetVariable(vp->GetCurrentTimestep(), vp->GetVariableName(), vp->GetRefinementLevel(), vp->GetCompressionLevel(), minExt, maxExt);
```

Comment:
The sole purpose of this grid is to get the grid type (Regular vs Curvilinear etc.). Is it possible to get this information without creating a new grid?

Reply:
There is not a method on the DataMgr to returns the grid type. One could of course be added if we thought it was worth the effort. In this particular instance i'm not sure it would buy us much as the data will have already been read and cached.